frontend/src/metabase/dashboard/actions/data-fetching.js

```diff
@@ -108,10 +108,7 @@ const loadingComplete = createThunkAction(
         dispatch(setShowLoadingCompleteFavicon(false));
       }, 3000);
     } else {
-      const dashboard = getDashboardComplete(getState());
-      const message = dashboard.is_app_age
-        ? t`Your page is ready`
-        : t`Your dashboard is ready`;
+      const message = t`Your dashboard is ready`;
       dispatch(setDocumentTitle(message));
```

Comment:
nit: think there's no point in the `message` variable anymore and we should be able to just pass it to `setDocumentTitle`